File: 프로그래머스/lv1/12926. 시저 암호/시저 암호.java
class Solution {
    public String solution(String s, int n) {
        String answer = "";
        char[] arr = s.toCharArray();
        for (int i = 0; i < arr.length; i++) {
            if (arr[i] == ' ') {
                answer += " ";
            }  else {
                int a = (int) arr[i];
                if (a >= 65 && a <= 90) {
                    a += n;
                    if (a > 90 && a < 123) {
                    a -= 26;
                }
                } else {
                    a += n;
                    if (a > 122) {
                        a -= 26;
                    }
                }
                char b = (char) a;
                answer += b;
            }
        }
        return answer;
    }
}

File: 프로그래머스/1/12926. 시저 암호/시저 암호.py
def solution(s, n):
    answer = ''
    lst = list(s)
    for i in lst:
        if i == " ":
            answer += " "
            continue
        if 65 <= ord(i) <= 90:
            if ord(i)+n > 90:
                answer += chr(ord(i)+n-26)
            else:
                answer += chr(ord(i)+n)
        elif 97 <= ord(i) <= 122:
            if ord(i)+n > 122:
                answer += chr(ord(i)+n-26)
            else:
                answer += chr(ord(i)+n)
    return answer